### PR TITLE
docs: update README to include note on decoupled model responses

### PR DESCRIPTION
**Pull request by mattwittwer**
#### What does the PR do?

Adding a note to clarify the collection of results from decoupled model requests. This has caused some confusion since this is different from response collection using '.get()' from fixed length responses. 

#### Checklist
- [x] PR title reflects the change and is of format `<commit_type>: <Title>`
- [x] Changes are described in the pull request.
- [x] Related issues are referenced.
- [x] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [ ] Added [test plan](#test-plan) and verified test passes.
- [ ] Verified that the PR passes existing CI.
- [x] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [ ] build
- [ ] ci
- [x] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

#### Where should the reviewer start?

Review the added note and the links to decoupled models and example code.

#### Test plan:

- CI Pipeline ID:

### CODE REVIEW

**Review by rmccorm4 — APPROVED** [review]

**Review by oandreeva-nv — APPROVED** [review]
lgtm

File: server/README.md
Comment (rmccorm4) [inline server/README.md:1]: oops: Update Copyright at top 2022-2024

Comment (rmccorm4) [reply server/README.md:1]: Caught by the checklist:
- [ ] Verified copyright is correct on all changed files.

Comment (mattwittwer) [reply server/README.md:1]: Updated copyright

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**